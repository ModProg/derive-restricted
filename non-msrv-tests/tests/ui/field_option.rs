#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone; T)]
struct Test1<T>(#[derive_where(option)] T);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone; T)]
enum Test2<T> {
	#[derive_where(option)]
	A(T),
}

fn main() {}
