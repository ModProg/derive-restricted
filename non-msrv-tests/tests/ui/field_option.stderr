error: unknown option
 --> tests/ui/field_option.rs:3:32
  |
3 | struct Test1<T>(#[derive_where(option)] T);
  |                                ^^^^^^

error: unknown option
 --> tests/ui/field_option.rs:8:17
  |
8 |     #[derive_where(option)]
  |                    ^^^^^^
